### PR TITLE
ENCM-38 ChIA QC

### PR DESCRIPTION
**Pull request by gabdank**

### CODE REVIEW

**Review by jenjou — COMMENTED** [review]
Looks OK, but small formatting request: can we add spaces after the colons? Some lines have a space, and some do not. `"description":"Fraction` vs `"comment": "Unique`
I think there are also typos in lines 49 "singe read", line 61 "containg", and possibly line 62 ("count."?)

File: src/encoded/schemas/chia_pet_alignment_quality_metric.json [hunk 1]
```diff
@@ -23,39 +23,51 @@
...
+            "description": "Non-redundant read pairs count containing a linker",
+            "comment": "PET count, after removing PCR duplicates from um_pet",
+            "type": "integer",
             "minimum": 0
         },
         "pet_red": {
             "title": "PET redundancy",
             "description": "Ratio of redundant PET to uniquely mapped PET",
-            "type":  "number",
+            "comment": "If PCR duplicated reads are high, then reads are highly redundant, and pet_red is closed to 1. If PCR duplicated reads are low, then the value decreases",
```
Comment (jenjou) [inline src/encoded/schemas/chia_pet_alignment_quality_metric.json]: Sorry, just noticed "closed to 1" -- probably "close to 1"?

File: src/encoded/schemas/chia_pet_alignment_quality_metric.json [hunk 2]
```diff
@@ -23,39 +23,51 @@
...
+            "type": "integer",
+            "description": "Total read pairs",
+            "comment": "It is number of read pairs from fastq files",
             "minimum": 0
         },
         "rp_bl": {
             "title": "Number of read pairs with bridge linker",
             "type":  "integer",
+            "description": "Read pairs with linker",
+            "comment": "We used 20bp our bridge-linker sequence, and allow 2bp mismatch to detect a linker sequence",
```
Comment (jenjou) [inline src/encoded/schemas/chia_pet_alignment_quality_metric.json]: I am not 100% sure about this, but is "20bp our" supposed to be "20bp for our"?

**Review by jenjou — CHANGES_REQUESTED** [review]
See comments

**Comment by jenjou** [comment]
I have additional questions/comments left above.

File: src/encoded/schemas/chia_pet_alignment_quality_metric.json (hunk 1 above)
Comment (gabdank) [reply src/encoded/schemas/chia_pet_alignment_quality_metric.json]: fixed

File: src/encoded/schemas/chia_pet_alignment_quality_metric.json (hunk 2 above)
Comment (gabdank) [reply src/encoded/schemas/chia_pet_alignment_quality_metric.json]: fixed

**Review by jenjou — APPROVED** [review]

**Pull request merged**